include/sparrow/types/data_traits.hpp

```diff
@@ -116,31 +116,31 @@ namespace sparrow
     {
         static constexpr data_type type_id = data_type::DURATION_SECONDS;
         using value_type = std::chrono::seconds;
-        using const_reference = std::chrono::seconds;
+        using const_reference = const std::chrono::seconds&;
     };
```

Comment:
I think this traits and the following can inherit from `common_native_type_traits`

Reply:
ok done